src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final String DEFAULT_FIGURE_COLOR = Color.WHITE.name();
+    public static final double DEFAULT_FIGURE_RADIUS = 10.0;
+    public static final String RANDOM_COLOR = new ColorSupplier().getRandomColor();
```

Comment:
make random and color supplier class field.
and get a random number and color when calling the method, otherwise it will always be the same color and number